### ISSUE TITLE
508-defect-4 [SCREENREADER]: Headings are nested improperly on main page

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21318]
Label: 20132

    - [508-defect-4]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4

Issue Title: Main page, headings nested improperly
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect to navigate by heading using the nesting levels to effectively scan content.
<h2>Details</h2>
On the main page, the heading "In the Spotlight..." is a level 3 heading. Since it represents a change in subject, it should be a level 2 heading as that is the level assigned to new topics throughout the page.
Acceptance Criteria

<h2>Environment</h2>
Consistent across all environments.
Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
